### PR TITLE
Updated logo

### PR DESCRIPTION
**Pull request by priyanshi1282**
## 👨‍💻 Changes proposed
Updated the Logo of Zipify.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
screenshot

### CODE REVIEW

File: views/components/header.ejs [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <nav class="navbar">
       <div class="logo">
 <img src="./logo.png" alt="logo">
+<!-- <img src="./logo.png" alt="logo"> -->
```
Comment (pulkit-30) [inline views/components/header.ejs:16]: Delete this

File: views/index.ejs [hunk 2]
```diff
@@ -15,6 +15,7 @@
 </div>
 <div class="flex">
   <img src="./logo-black.png" alt="logo-black" class="logo-black" />
+  <!-- <img src="./logo-black.png" alt="logo-black" class="logo-black" /> -->
```
Comment (pulkit-30) [inline views/index.ejs:18]: Delete this !

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by priyanshi1282** [comment]
Hey @pulkit-30 I did required changes.

**Review by pulkit-30 — APPROVED** [review]
Great Thanks 🎉

**Comment by priyanshi1282** [comment]
> Great Thanks 🎉

It was nice working with you 😀

**Comment by priyanshi1282** [comment]
Hey @pulkit-30, @Ayush7614  I think you have to add level0/1/2/3 label also so that I can get points for this PR.

**Pull request merged**